components/squad/squad.lua

```diff
@@ -0,0 +1,76 @@
+local Class = require('Module:Class')
+local Arguments = require('Module:Arguments')
+local String = require('Module:String')
+
+local Squad = Class.new()
+
+function Squad:init(frame)
```

Comment:
Could move init into the constructor

Reply:
Not with the new variable usage of Squad.TYPE_ACTIVE etc